### PR TITLE
Add tool to request mission details and print them.

### LINKED ISSUE #148: Add interface to request mission details
E.g. fly zones, waypoints, etc

### PR DESCRIPTION
**Pull request by pmtischler**
Fixes #148

### CODE REVIEW

File: client/tools/README.md [hunk 1]
```diff
@@ -27,5 +26,17 @@ Limitations:
...
+
+prober.py
+---------
+
+The prober is a simple use of the interoperability client which validates the
+system is working and recieving data. Assuming tests passed before deployment,
+this can be used to continuously validate the system is accessible by teams.
+
+```
+python prober.py [interop_server_host] [interop_time] [username] [password] [generator] [flightsim_kml_path]
```
Comment (prattmic) [inline client/tools/README.md]: prober.py is executable, so just say `./prober.py` like the other examples.

Comment (pmtischler) [inline client/tools/README.md]: Done.

File: client/tools/get_missions.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+from interop import Client
+from interop import Target
+
+
+def main(url, username, password):
+    """Program to get and print the mission details."""
+    client = Client(url, username, password)
+    missions = client.get_missions()
+    print missions
```
Comment (prattmic) [inline client/tools/get_missions.py]:
```py
from __future__ import print_statement
import pprint

for i, m in enumerate(missions):
    print("Mission %d" % i)
    pprint.pprint(m.serialize())
```

Should make this much easier to read and compatible with Python 3.

**Comment by prattmic** [comment]
One suggestion about making the output easier to read and Python 3 compatibility. Otherwise LGTM

File: client/tools/get_missions.py (hunk 2 above)
Comment (prattmic) [inline client/tools/get_missions.py]: Hm, apparently Mission doesn't support serialize. I'd still recommend some kind of custom printing logic. The output must be nigh unreadable as-is, given the amount of stuff in Mission.

Comment (pmtischler) [inline client/tools/get_missions.py]: I was relying on the string encoding of Mission. I haven't implemented serialization for Mission yet. I've changed it to use print() though.

**Comment by pmtischler** [comment]
Filed #170 to handle the serializing & pretty print. I"m pushing off any work that is non blocking or not needed for team testing until after the rules are released.

File: client/tools/get_missions.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# Program to request and print mission details.
+
+import argparse
+import getpass
+from __future__ import print_statement
```
Comment (prattmic) [inline client/tools/get_missions.py]: It is print_function, not print_statement, I was dumb. Statement is the old behavior :)

**Pull request merged**